src/main/java/edu/ucsb/cs156/gauchoride/entities/IndividualRides.java

```diff
@@ -0,0 +1,30 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "individualRides")
+public class IndividualRides {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private Long driverId;
+  private Long[] riderIds;
```

Comment:
I'm still confused about this being an array.

How will will this work in the database?

Reply:
I checked the database.  It stores this as a "byte array".  That's not going to do what we want.

I suggest that you change this to simply 
```suggestion
  private Long riderId;
```

And make the assumption of one rider per ride for now; and then we can leave multiple riders as work for the summer 2023 students.